.gitignore

```diff
@@ -81,4 +81,8 @@ artifacts
 .openzeppelin
 
 # .env file
-.env
\ No newline at end of file
+.env
+
+package-lock.json
```

Comment:
Reason: It is because a `yarn.lock` file also exists and these are functionally the same file from different package managers (npm vs yarn).

Resolution: We will go with `yarn` as our package manager. As such, adding the `package-lock.json` file to gitignore is necessary to avoid resolution inconsistencies caused by unsynchronized lock files.